README.md

```diff
@@ -34,13 +34,13 @@ To see FalkorDB in action, visit [Demos](https://github.com/FalkorDB/FalkorDB/tr
 To quickly try out FalkorDB, launch an instance using docker:
 
 ```
-docker run -p 6379:6379 -it --rm falkordb/falkordb:edge
+docker run -p 6379:6379 -it --rm -v ./data:/data falkordb/falkordb:edge --dir /data
 ```
 
 Or, to use the built-in browser-based interface, run:
 
 ```
-docker run -p 6379:6379 -p 3000:3000 -it --rm falkordb/falkordb:edge
+docker run -p 6379:6379 -p 3000:3000 -it --rm -v ./data:/data falkordb/falkordb:edge --dir /data
```

Comment:
Thank you,
I've tried running this command locally and I've found the following in my host `data` folder:
```
README.md dump.rdb  go.md
```

`dump.rdb` is expected, but `README.md` and `go.md` aren't. can we maintain a clear separation, between the RDB and the rest of the content ?

Reply:
I'm not sure why you see these extra files. Did they exist before?

Reply:
No they didn't